File: Basketball One-on-One/basketballoneonone.c
#include <stdio.h>
#include <string.h>

int main() {
    char record[201];
    scanf("%s", record);

    int scoreA = 0, scoreB = 0;

    for (int i = 0; i < strlen(record); i += 2) {
        int points = record[i + 1] - '0';

        if (record[i] == 'A') {
            scoreA += points;
        } else {
            scoreB += points;
        }
    }

    if (scoreA > scoreB) {
        printf("A");
    } else {
        printf("B");
    }

    return 0;
}

File: Basketball One-on-One/basketballoneonone.java
import java.util.Scanner;

public class basketballoneonone {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String record = scanner.next();

        int scoreA = 0, scoreB = 0;

        for (int i = 0; i < record.length(); i += 2) {
            int points = Character.getNumericValue(record.charAt(i + 1));

            if (record.charAt(i) == 'A') {
                scoreA += points;
            } else {
                scoreB += points;
            }
        }

        if (scoreA > scoreB) {
            System.out.println("A");
        } else {
            System.out.println("B");
        }
    }
}
